src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,104 @@
+- A backup of a virtual cluster can be restored as a virtual cluster in any storage cluster with cluster virtualization enabled.
+
+For more details about backing up and restoring a cluster with cluster virtualization enabled, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#disaster-recovery).
+
+{{site.data.alerts.callout_success}}
+For details about configuring and using Physical Cluster Replication for disaster recovery, refer to [Physical Cluster Replication Overview]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}).
+{{site.data.alerts.end}}
+
+### Observability
+
+When cluster virtualization is enabled, cluster log messages and metrics are scoped to the virtual cluster or to the system interface, and are labeled accordingly.
+
+When connected to a virtual cluster, most pages and views are scoped to the virtual cluster. By default the DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system interface. To allow the DB Console to display system-level metrics from within a virtual cluster, you can grant the virtual cluster the `can_view_node_info` permission.
+
+Some pages and views are only viewable from the system interface, including those pertaining to overall cluster health.
+
+For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#observability)
+
+### SQL API
+
+When cluster virtualization is enabled, certain low-level SQL APIs (TODO: Which ones) are accessible only by those users with the `admin` role on the system interface, and not from virtual clusters.
+
+### Span Config Bounds
+
+- Span config bounds can be set in a virtual cluster or in the system interface. Span config bounds set in the system interface override zone config settings that are set in a virtual cluster.
+- Zone configs can be set only in a virtual cluster, but span config bounds set in the system interface override zone configs that are set in a virtual cluster.
+
+## See also
+
+- [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %})
+- [Cluster Setting Scopes]({% link {{ page.version.version }}/cluster-virtualization-setting-scopes.md %})
+- [Cluster Metric Scopes]({% link {{ page.version.version }}/cluster-virtualization-metric-scopes.md %})
+- [Physical Cluster Replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %})
+
+## Known Limitations
+
+In CockroachDB {{page.version.version}}, cluster virtualization has the following limitations:
+
+- Cluster virtualization is supported only for [Physical Cluster Replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}). General-purpose virtual clusters are not supported.
+- A single physical cluster can have a maximum of one system interface and one virtual cluster.
```

Comment:
Nit: it's impossible to create more than one system interface, so I think we can leave that part out.

I also think that the previous line is unnecessary.  We should support it for all use cases, even if PCR is not being used (or not being used immediately).